es/cheatsheet-supervised-learning.md

```diff
@@ -1,567 +1,566 @@
 <br>
 
 **56. Gaussian Discriminant Analysis**
 
-&#10230;
+&#10230; Análisis discriminante Gaussiano
 
 <br>
 
 **57. Setting ― The Gaussian Discriminant Analysis assumes that y and x|y=0 and x|y=1 are such that:**
 
-&#10230;
+&#10230; Marco - El Análisis discriminante Gaussiano asume que y, x|y=0 y x|y=1 son de la siguiente forma:
 
 <br>
 
 **58. Estimation ― The following table sums up the estimates that we find when maximizing the likelihood:**
 
-&#10230;
+&#10230; Estimación - La siguiente tabla resume las estimaciones encontradas al maximizar la probabilidad:
 
 <br>
 
 **59. Naive Bayes**
 
-&#10230;
+&#10230; Naive Bayes
 
 <br>
 
 **60. Assumption ― The Naive Bayes model supposes that the features of each data point are all independent:**
 
-&#10230;
+&#10230; Supuestos - El modelo Naive Bayes supone que las características de cada punto de los dato son todas independientes:
 
 <br>
 
 **61. Solutions ― Maximizing the log-likelihood gives the following solutions, with k∈{0,1},l∈[[1,L]]**
 
-&#10230;
+&#10230; Soluciones - Maximizar la log-probabilidad da las siguientes soluciones, con k∈{0,1},l∈[[1,L]]
 
 <br>
 
 **62. Remark: Naive Bayes is widely used for text classification and spam detection.**
 
-&#10230;
+&#10230; Observación: Naive Bayes es comunmente utilizado para la clasificación de texto y la detección de correo no deseado (spam).
 
 <br>
 
 **63. Tree-based and ensemble methods**
 
-&#10230;
+&#10230; Métodos basados en árboles y conjuntos
 
 <br>
 
 **64. These methods can be used for both regression and classification problems.**
 
-&#10230;
+&#10230; Estos métodos pueden ser utilizados tanto en problemas de regresión como clasificación.
 
 <br>
 
 **65. CART ― Classification and Regression Trees (CART), commonly known as decision trees, can be represented as binary trees. They have the advantage to be very interpretable.**
 
-&#10230;
+&#10230; CART - Árboles de clasificación y regresión (en inglés, Classification and Regression Trees) (CART), comunmente conocidos como árboles de desición, pueden ser representados como árboles binarios. Presentan la ventaja de ser muy interpretables.
 
 <br>
 
 **66. Random forest ― It is a tree-based technique that uses a high number of decision trees built out of randomly selected sets of features. Contrary to the simple decision tree, it is highly uninterpretable but its generally good performance makes it a popular algorithm.**
 
-&#10230;
+&#10230; Bosques aleatórios (en inglés Random forest) - Es una téctica basada en árboles que utiliza una gran cantidad de árboles de desición cionstruidos a partir de conjuntos de características seleccionadas al azar. A diferencia del árbol de desición simple, la solición del método de bosques aleatórios es dificilmente inrerpretable aunque por su frecuente buen rendimiento es un algoritmo muy popular.
```

Comment:
"decisión" instead of "desición", and "construidos"  instead of "cionstruidos", just accent in difícilmente